TheraLang.Web/ClientApp/src/app/app-routing.module.ts

```diff
@@ -33,11 +33,7 @@ const routes: Routes = [
         component: ProjectFormComponent,
         canActivate: [AuthGuard]
       },
-      {
-        path: "projects/:id",
-        component: ProjectInfoComponent,
-        canActivate: [AuthGuard]
-      },
+      { path: "projects/:id", component: ProjectInfoComponent},
```

Comment:
Why did you delete CanActivate fucntion ?

Reply:
Cuz guest will can visit to details page